ISSUE_TEMPLATE/team_ticket_internal.yml

```diff
@@ -0,0 +1,33 @@
+name: Ticket (Internal Use Only)
+description: Tracking our work through tickets
+title: "[Ticket]: "
+labels: ["backlog"]
```

Comment:
So this label is a remnant from when I set up an automation in Github Projects to put anything labeled `backlog` into our project in the backlog column. I'm happy for it to be whatever! 

I don't believe it's possible to add non-static labels via form input at this time, but I can check further. The output is markdown and you can add whatever labels you'd like in the same ways you would on any issue via the side-panel wizard. Is there a static label you'd like added by default?

Reply:
Cool, I figured I might be asking too much with label suggestions. Since the internal team will have the ability to select labels once it's been created, I'm not sure we need a default label unless we wanted to have a `task` label.

But I'd be fine with leaving it blank!